scripts/copy-php-beta-migration-models.ps1

```diff
@@ -0,0 +1,12 @@
+# Removes all models under src/ excluding all .php files in the root directory
```

Comment:
reuse https://github.com/microsoftgraph/MSGraph-SDK-Code-Generator/blob/dev/scripts/copy-php-beta-models.ps1 and make the relative output path a parameter instead